CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -82,6 +55,55 @@ class SchedulingPlacesViewController: UIViewController {
     private var selectedCampusLocations = [String]()
     private var selectedCtownLocations = [String]()
 
+    private var isOtherSide: Bool
+    private var confirmedLocation: String!
+
+    private var availabilities: [String: [String]] = [:]
+    private var confirmedTime: [String: String] = [:]
+
+    private let headerHeight: CGFloat = 50
+    private let interitemSpacing: CGFloat = 12
+    private let lineSpacing: CGFloat = 12
+
+    // TODO: Replace with networking when available
+    private var campusLocations = [
+        "Atrium Cafe",
+        "Cafe Jennie",
+        "Gimme Coffee",
+        "Goldie's Cafe",
+        "Green Dragon",
+        "Libe Cafe",
+        "Mac's Cafe",
+        "Martha's Cafe",
+        "Mattin's Cafe",
+        "Temple of Zeus"
+    ]
+    private var ctownLocations = [
+        "Kung Fu Tea",
+        "Starbucks",
+        "Mango Mango",
+        "U Tea"
+    ]
+    private let matchLocations = [
+        "Goldie's Cafe",
+        "Green Dragon",
+        "Kung Fu Tea"
+    ]
+
+    init(isOtherSide: Bool, availabilities: [String: [String]], confirmedTime: [String: String]) {
+        self.isOtherSide = isOtherSide
+        if isOtherSide {
+            self.confirmedTime = confirmedTime
```

Comment:
could we rename to `confirmedTimes`? i mgiht be understanding it wrong but it seems like multiple times rather than just one time object